distribution/test/itests/test-itests-catalog/src/test/java/ddf/catalog/test/TestSecurity.java

```diff
@@ -170,6 +185,12 @@ public void beforeTest() throws Exception {
         waitForAllBundles();
         configurePDP();
         waitForHttpEndpoint(SERVICE_ROOT + "/catalog/query");
+        keystore = KeyStore.getInstance("jks");
```

Comment:
What is the keystore getting used for? I don't see it being used anywhere. Same with the truststore